packages/nimble-components/src/table-column/enum-text/cell-view/index.ts

```diff
@@ -0,0 +1,52 @@
+import { DesignSystem } from '@microsoft/fast-foundation';
```

Comment:
We haven't been consistent on it for the cell view and group header view elements but every element should have those trivial registration check tests:

https://github.com/ni/nimble/blob/158b1d29e9cb8d6591662404ee4ed0921b3e1361/packages/nimble-components/src/breadcrumb/tests/breadcrumb.spec.ts#L3-L13

For the elements outside this PR could you capture those in a tech debt task and make sure we have them in a follow-up PR?

Reply:
I went ahead and added them in this PR.

Reply:
I guess that's fine for this PR since it's already done but really based on the history of this PR and the discussions we have been having for smaller PRs I would not recommend that in the future.